src/components/EntityInfo.vue

```diff
@@ -1,129 +1,138 @@
 <template>
   <div id="entity-info" :class="{ 'staff': isRoleStaff }">
-    <v-container>
-      <v-breadcrumbs :items="breadcrumbs" divider=">" class="breadcrumb mb-5">
+    <v-container class="py-2 pb-6">
+      <v-breadcrumbs :items="breadcrumbs" divider=">" class="pa-0">
         <v-breadcrumbs-item
           slot="item"
           slot-scope="{ item }"
           exact
           :to="item.to"
-          :href="item.href">
+          :href="item.href"
+        >
           {{ item.text }}
         </v-breadcrumbs-item>
       </v-breadcrumbs>
 
       <!-- Entity Name, Entity Status -->
-      <div class="title-container">
-        <div v-if="businessId" class="mb-1" id="entity-legal-name" aria-label="Business Legal Name">
-          <span>{{ entityName || 'Unknown Name' }}</span>
-        </div>
-        <div v-if="tempRegNumber" class="mb-1" id="incorp-app-title" aria-label="Incorporation Application Title">
-          <span>{{ entityName || entityTypeToNumberedDescription(entityType)}}</span>
-        </div>
-
-        <v-chip v-if="isGoodStanding" class="blue" id="entity-status" small label text-color="white">
-          <span>In Good Standing</span>
-        </v-chip>
-        <v-chip v-else-if="isPendingDissolution" class="red" id="entity-status" small label text-color="white">
-          <span>Pending Dissolution</span>
-        </v-chip>
-        <v-chip v-else-if="isNotInCompliance" class="red" id="entity-status" small label text-color="white">
-          <span>Not in Compliance</span>
-        </v-chip>
-      </div>
-
-      <!-- Business Numbers, Contact Info -->
-      <div class="business-info" v-if="businessId">
-        <div class="business-info__meta">
-          <dl>
-            <dt>Business No:</dt>
-            <dd class="ml-2" id="entity-business-number">
-              <span>{{ entityBusinessNo || 'Not Available' }}</span>
-            </dd>
-
-            <dt>Incorporation No:</dt>
-            <dd class="ml-2" id="entity-incorporation-number">
-              <span>{{ entityIncNo || 'Not Available' }}</span>
-            </dd>
-          </dl>
+      <div class="d-flex justify-space-between mt-5">
+        <div class="left-column align-self-end">
```

Comment:
Slight layout change + cleanup.
Hide whitespace changes to see this more cleanly.